### ISSUE TITLE
Compilation error / Vaadin 24.0.0.beta1

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/TwinColGridAddon#133]
```
/src/test/java/com/flowingcode/vaadin/addons/twincolgrid/OrientationDemo.java:[59,56] 
cannot infer type arguments for com.vaadin.flow.component.select.Select<>
```

**Comment by javier-godoy** [comment]
Demo uses the deprecated constructor removed in https://github.com/vaadin/flow-components/pull/4400

**Comment by javier-godoy** [comment]
Fixed in v3.0.0
